tests/checks/integration/test_elastic.py

```diff
@@ -297,3 +346,87 @@ def test_health_event(self):
             tags=['host:localhost', 'port:9200'],
             count=1
         )
+
+    def test_pshard_metrics(self):
```

Comment:
Since you're testing for new metrics you should add a coverage_report at the end of this method.